components/console-backend-service/internal/domain/k8s/namespace_resolver_test.go

```diff
@@ -4,45 +4,76 @@ import (
 	"errors"
 	"testing"
 
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlerror"
+
 	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/k8s"
 	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/k8s/automock"
 	appAutomock "github.com/kyma-project/kyma/components/console-backend-service/internal/domain/shared/automock"
-	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlerror"
 	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestNamespaceResolver_NamespacesQuery(t *testing.T) {
 	t.Run("Success", func(t *testing.T) {
```

Comment:
Do I see correctly that there is no test for a subscription? Please add it. Remember to check if all is cleaned up when the context is closed. Otherwise, we would have routines leakage.